drivers/pinctrl/pinctrl_gecko_s1.c

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) 2023 Silicon Labs
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/logging/log.h>
+
+#include <soc_gpio.h>
+
+#define DT_DRV_COMPAT silabs_gecko_s1_pinctrl
```

Comment:
this file is not using any `DT_INST` api, so this can be removed

Reply:
Done.